### PR TITLE
upgrade gatk version (resolves #229)

### LINKED ISSUE #229: upgrade version of gatk
updating to latest version of gatk: 3.7

### PR DESCRIPTION
**Pull request by tpesout**
updated to latest: 3.7

### CODE REVIEW

File: gatk/build/Dockerfile [hunk 1]
```diff
@@ -3,19 +3,20 @@ FROM java:8-jdk
...
 
 RUN apt-get update && apt-get install -y \
+    wget \
 	git \
 	maven \
 	default-jdk
 
 # Create a new source directory
-WORKDIR /home
+WORKDIR /home/
```
Comment (jvivian) [inline gatk/build/Dockerfile]: This change needed?

File: gatk/build/Dockerfile [hunk 2]
```diff
@@ -24,7 +25,11 @@ RUN git checkout 3.5
 RUN find . -name "*.java" -exec sed -i -e "s/import oracle.jrockit.jfr/\/\/import oracle.jrockit.jfr/g" {} \;
 
 # Build GATK
+# move file and remove source folder in same command (downloads lots of dependencies)
 RUN mvn -Ddisable.queue install
+RUN ls /home/gatk-protected/gatk-protected-3.7/
```
Comment (jvivian) [inline gatk/build/Dockerfile]: Why are you running `ls` here?

File: gatk/build/Dockerfile [hunk 3]
```diff
@@ -24,7 +25,11 @@ RUN git checkout 3.5
...
+RUN ls /home/gatk-protected/gatk-protected-3.7/target/
```
Comment (jvivian) [inline gatk/build/Dockerfile]: same

File: gatk/build/Dockerfile [hunk 4]
```diff
@@ -24,7 +25,11 @@ RUN git checkout 3.5
...
+RUN mv /home/gatk-protected/gatk-protected-3.7/target/GenomeAnalysisTK.jar /home/gatk-protected/gatk.jar
 
 # Move jar to current WORKDIR
-RUN mv /home/gatk-protected/target/GenomeAnalysisTK.jar /home/gatk-protected/gatk.jar
+WORKDIR /home/gatk-protected
```
Comment (jvivian) [inline gatk/build/Dockerfile]: By convention the final `WORKDIR` is set to **/data**

File: gatk/build/Dockerfile [hunk 5]
```diff
@@ -3,19 +3,20 @@ FROM java:8-jdk
 MAINTAINER John Vivian, jtvivian@gmail.com
 
 RUN apt-get update && apt-get install -y \
+    wget \
 	git \
```
Comment (jvivian) [inline gatk/build/Dockerfile]: Can remove `git` since you're not using it anymore

**Review by jvivian — CHANGES_REQUESTED** [review]

File: gatk/build/Dockerfile [hunk 6]
```diff
@@ -24,7 +25,11 @@ RUN git checkout 3.5
 RUN find . -name "*.java" -exec sed -i -e "s/import oracle.jrockit.jfr/\/\/import oracle.jrockit.jfr/g" {} \;
```
Comment (fnothaft) [inline gatk/build/Dockerfile]: Is this line still needed in 3.7?

**Review by fnothaft — CHANGES_REQUESTED** [review]

File: gatk/build/Dockerfile (hunk 6 above)
Comment (tpesout) [reply gatk/build/Dockerfile]: unfortunately yes :/

Comment (jvivian) [reply gatk/build/Dockerfile]: lololol

Comment (fnothaft) [reply gatk/build/Dockerfile]: thats hot

**Review by jvivian — APPROVED** [review]

**Review by jvivian — CHANGES_REQUESTED** [review]
Makefile tag needs updating to reflect version change.

**Review by fnothaft — APPROVED** [review]

**Review by jvivian — APPROVED** [review]

**Comment by jvivian** [comment]
jenkins, test this please

**Pull request merged**